examples/stm32/xively_demo_stm32f2xx_nucleo_eth/readme.md

```diff
@@ -0,0 +1,14 @@
+Example Project Status
+======================
+
+This example is broken for the latest version of the Cube SDK. It was proven
+working with version `STM32Cube_FW_F4_V1.14.0`, but broke at
+`STM32Cube_FW_F4_V1.16.0`
+
+We don't plan on maintaining the example for every SDK release. We're not
+removing it either, in case it's useful to developers looking to use this
+platform.
+
+You should expect a significant amount of work to get it working. An SDK update
```

Comment:
I wouldn't guess how much work is needed to get it working. Maybe it is super easy for a STM expert, don't know.